src/Config.js

```diff
@@ -1,4 +1,12 @@
 class Config {
+  static get sinopiaServerBase() {
+    return process.env.SINOPIA_SERVER_BASE || 'http://localhost:8080'
```

Comment:
See [comment](https://github.com/LD4P/sinopia_editor/issues/458#issuecomment-484235571) about adding **SINOPIA_SERVER_BASE** to Terraform.